README.md

```diff
@@ -4,7 +4,7 @@ Nextcloud server packaged as a snap. It consists of:
 
 - Nextcloud 22
 - Apache 2.4
-- PHP 7.4
+- PHP 8
```

Comment:
Maybe PHP `8.0` would be better here, I think people are interested in the particular point release used, and 8.1 already has a RC.